src/DebugEngineHost/HostNatvisProject.cs

```diff
@@ -207,6 +237,26 @@ public static string FindSolutionRootImpl()
                 return root;
             }
 
+            private static void BuildEnvironmentPath(string name, IVsExtensionManagerPrivate pem, List<string> paths)
+            {
+                pem.GetEnabledExtensionContentLocations(name, 0, null, null, out var contentLocations);
+                if (contentLocations > 0)
+                {
+                    var rgStrings = new string[contentLocations];
+                    var rgbstrContentLocations = new string[contentLocations];
+                    var rgbstrUniqueStrings = new string[contentLocations];
+
+                    var hr = pem.GetEnabledExtensionContentLocations(name, contentLocations, rgbstrContentLocations, rgbstrUniqueStrings, out var actualContentLocations);
+                    if (hr == VSConstants.S_OK && actualContentLocations > 0)
+                    {
+                        for (var i = 0; i < actualContentLocations; ++i)
+                        {
+                            paths.Add(rgbstrContentLocations[i]);
+                        }
```

Comment:
nit: any reason to not just return this array instead of adding it to a list?

Reply:
works cleaner with caller as is, especially now that caller is the new "FindNatvis".

Reply:
In that case: switch to `paths.AddRange(rgbstrContentLocations);`